sacred/experiment.py

```diff
@@ -62,7 +64,13 @@ def __init__(self, name: Optional[str] = None,
             Optional full path to the base directory of this experiment. This
             will set the scope for automatic source file discovery.
 
+        additional_host_info : optional
+            Optional dictionary containing as keys the names of the pieces of
+            host info you want to collect, and as
+            values the functions collecting those pieces of information.
         """
+        self.additional_host_info = additional_host_info or {}
```

Comment:
```suggestion
        self._additional_host_info = additional_host_info or {}
```
We should probably make this "private" to make it explicit that this attribute should only be set through the constructor.

Reply:
I'm not sure it's needed. User should be allowed to access it, and setting an attribute in a third party library is discouraged in python except if mentioned explicitly in the documentation. If you feel strongly that it should be private, I'll change it though.